requirements-full.txt

```diff
@@ -3,3 +3,4 @@ scanimage-tiff-reader==1.4.1.4
 neuroconv[video]>=0.4.6  # Uses the VideoCaptureContext class
 natsort>=8.3.1
 isx>=1.0.4
+aicsimageio>=4.14.0
```

Comment:
From @h-mayorquin to check out for relevant info in CI testing:

Install wine:
https://github.com/h-mayorquin/spikeinterface/blob/9cd4d127f0db0b354114eda083d892131e070abb/.github/actions/install-wine/action.yml#L1-L22

https://github.com/h-mayorquin/spikeinterface/blob/9cd4d127f0db0b354114eda083d892131e070abb/.github/workflows/full-test.yml#L131-L139

https://github.com/h-mayorquin/spikeinterface/blob/9cd4d127f0db0b354114eda083d8921[…]70abb/src/spikeinterface/extractors/tests/test_neoextractors.py